cg/models/downsample/downsample_data.py

```diff
@@ -100,6 +100,8 @@ def _generate_statusdb_downsampled_sample_record(
             priority=Priority.standard,
             customer=self.original_sample.customer,
             application_version=application_version,
+            received=self.original_sample.received_at,
+            prepared_at=self.original_sample.prepared_at,
```

Comment:
do we also want to set last_sequenced_at?

Reply:
@Vince-janv I thought downsampled samples are due for analysis. If that is not the case, we should set it I guess?

Reply:
They are due for bioinformatic analysis, but will not be sequenced. (Downsampling is basically just randomly sampling reads from FASTQ files until you have a set amount). IE we create a "smaller" sample of one that has already been sequenced

Reply:
But you know the counting logic better than I do so it's up to you wether we need to set it or not

Reply:
Then it makes sense to set the value, otherwise we will always have it in sequencing on top of any TB status.

Reply:
Added the value